.solhint.json

```diff
@@ -2,13 +2,13 @@
   "extends": "solhint:recommended",
   "plugins": ["prettier"],
   "rules": {
-    "code-complexity": ["error", 8],
+    "code-complexity": ["error", 9],
     "compiler-version": ["error", ">=0.8.23"],
     "const-name-snakecase": "off",
     "no-empty-blocks": "off",
     "constructor-syntax": "error",
     "func-visibility": ["error", { "ignoreConstructors": true }],
-    "max-line-length": ["error", 120],
+    "max-line-length": ["error", 121],
```

Comment:
It's because of this line at the moment
<img width="761" alt="image" src="https://github.com/user-attachments/assets/4d0d5b8a-cab3-4ede-b416-c52b678d27bf">

Reply:
I see. It would be better to update it to multiple lines to follow the convention and maintain consistency with Solidity code style: https://docs.soliditylang.org/en/latest/style-guide.html#maximum-line-length

Reply:
addressed in https://github.com/storyprotocol/protocol-core-v1/pull/215/commits/3bd956f92ca20b433625c27f1e4ad349176a1352